qiita_db/portal.py

```diff
@@ -0,0 +1,53 @@
+from .sql_connection import SQLConnectionHandler
+from .util import convert_to_id
+from qiita_core.qiita_settings import qiita_config
+
+# Only make functions available for main portal
+if qiita_config.portal == "QIITA":
+    def get_studies_by_portal(portal):
```

Comment:
Given the nature of the functions here, it looks like it makes sense to create a `Portal` object. This will allow us to create portals from the "Admin" Qiita interface by calling `Portal.create(name)`. The functions here will then become:
 - `get_studies_by_portal` -> property `studies`
 - `add_study_to_portal` -> `add_study`
 - `remove_study_from_portal` -> `remove_study`